src/coordinates/uniform_cartesian.hpp

```diff
@@ -318,6 +318,7 @@ class UniformCartesian {
   const std::array<Real, 3> &GetXmin() const { return xmin_; }
   const std::array<int, 3> &GetStartIndex() const { return istart_; }
   const char *Name() const { return name_; }
+  static const char *StaticName() { return name_; }
```

Comment:
What's up with this?

Reply:
somewhere `block_list[0].coords.Name()` was called in the HDF5 output code. When I was trying to get that to work, I switched to a static method. This can go before the PR gets merged.